lib/index.js

```diff
@@ -76,7 +76,7 @@ function downloadFromInfoCallback(stream, info, options) {
       url += '&begin=' + util.fromHumanTime(options.begin);
     }
     doDownload(stream, url, options, {
-      trys: 5,
+      trys: options && options.retries || 5,
```

Comment:
No need to check if `options` is defined, that is checked above at the start of the function, and set to `{}` if it's not set.